src/App.js

```diff
@@ -1,46 +1,36 @@
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import ChatOverview from "./screens/ChatOverview";
+import PopUpContactPage from "./screens/PopUpContactPage";
+import ChildOverviewPage from "./screens/ChildOverviewPage";
+import Parse from "parse/dist/parse.min.js";
+import LogIn from "./screens/LogIn";
+import SignUpChild from "./screens/SignUpChild";
+import SignUpParent from "./screens/SignUpParent";
+import Contacts from "./screens/Contacts";
 
-import logo from './logo.svg';
-import ButtonPurple from './components/buttonPurple';
-import ButtonYellow from './components/buttonYellow';
-import Input from './components/LoginInput';
-import React from "react";
-import "./App.css";
-import Sidebar from "./components/Sidebar/Sidebar.js";
-import Topbar from "./components/Topbar/Topbar.js";
-import TextField from "./components/TextField.js";
+// Parse initialization configuration
```

Comment:
Consider moving these constants to a separate file named config.js (or similar) or even better in an environment (.env) file